clang/tools/driver/cc1_main.cpp

```diff
@@ -198,13 +199,20 @@ static int PrintSupportedExtensions(std::string TargetStr) {
   std::unique_ptr<llvm::TargetMachine> TheTargetMachine(
       TheTarget->createTargetMachine(TargetStr, "", "", Options, std::nullopt));
   const llvm::Triple &MachineTriple = TheTargetMachine->getTargetTriple();
+  const llvm::MCSubtargetInfo *MCInfo = TheTargetMachine->getMCSubtargetInfo();
+  const llvm::ArrayRef<llvm::SubtargetFeatureKV> Features =
+    MCInfo->getAllProcessorFeatures();
+
+  llvm::StringMap<llvm::StringRef> llvmDescMap;
```

Comment:
I don't see the need for the `llvm` prefix here, I guess you're saying that these come from llvm as in the backend, which is true but not really useful information here.

`DescMap` would be fine instead.

Reply:
It indicate the origin from llvm backend.

Reply:
Sure that's got some logic to it, but the code is clearly building a map with data from MCInfo, so it's implicit that it came from the llvm backend. And there isn't another source of data, so it's not like we have an llvm description map and a clang description map and need to know the difference.

Do as you wish it's not a big deal, but if you keep it I think the name should be `LLVMDescMap` to match https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly.

Reply:
I'll leve it as sugessted ```DescMap```.